spoon-pom/pom.xml

```diff
@@ -274,12 +274,14 @@
                     <configuration>
                         <header>LICENSE-short.txt</header>
                         <includes>
-                            <include>src/main/java/**</include>
```

Comment:
that results in deactivating license check (basedir is not src)

Reply:
Okay! Now I understand why I struggled so much... :-( My local copy was out of sync (I had basedir=src).